support/kind/kind.go

```diff
@@ -46,6 +46,28 @@ func (k *Cluster) WithVersion(ver string) *Cluster {
 	return k
 }
 
+func (k *Cluster) getKubeconfig() (string, error) {
```

Comment:
Suggestion: maybe cache the `kubeconfig` file name in type Cluster.  Then, only pull the kubeconfig if Cluster.kubeconfig == "", otherwise do pull it as done in this method.

Nevermind, I think `Cluster.getKubeconfig` is only called at creation. So that may not be an issue.

Reply:
Yes -- this was only happening in the Create + CreateWithConfig functions